app/src/main/java/org/mozilla/fenix/components/accounts/FenixAccountManager.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.accounts
+
+import android.content.Context
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import org.mozilla.fenix.ext.components
+
+/**
+ * Component which holds a reference to [FxaAccountManager]. Manages account authentication,
+ * profiles, and profile state observers.
+ */
+open class FenixAccountManager(context: Context) {
+    val accountManager = context.components.backgroundServices.accountManager
+    val authenticatedAccount = accountManager.authenticatedAccount() != null
+
+    /**
+     * Get the email address associated with the authenticated account profile. Returns null if
+     * the account is not authenticated or the email address is null.
+     */
+    fun getAuthAccountEmail(): String? {
+        val email = accountManager.accountProfile()?.email
+        return if (authenticatedAccount && !email.isNullOrEmpty()) email else null
```

Comment:
We could simplify this function by just returning directly `accountManager.accountProfile()?.email` as we are not going to have a profile if we are not authenticated and either way we are returning `String?`, this means that the consumer have to do the null-able or empty validation.

Reply:
We _could_ have a profile when the account needs reauthentication. I guess in that case it might not matter if we show the email?